src/directions/types.ts

```diff
@@ -269,7 +269,7 @@ export const MapLibreGlDirectionsDefaultConfiguration: Omit<MapLibreGlDirections
   dragThreshold: 10,
 };
 
-export type PointType = "WAYPOINT" | "SNAPPOINT" | "HOVERPOINT";
+export type PointType = "WAYPOINT" | "SNAPPOINT" | "HOVERPOINT" | string;
```

Comment:
Can you share why you make this type broader?

Reply:
Good point. I've been working on my another project recently and realized that in order to provide a proper extensibility-through-subclass support these 3 standard built-in point types are not enough. Sometimes you need to implement custom points. So these 3 are left as they were in order to provide IDE-autocompletion basically, and the `string` type broadens things up to provide a capability to create points of different types.